docs/SUMMARY.md

```diff
@@ -1,5 +1,6 @@
 # Summary
 
+- [Contributing](contributing/README.md)
```

Comment:
Why does this link to `contributing/README.md` which seems to redirect to `../../CONTRIBUTING.md` instead of linking to `../CONTRIBUTING.md` from here? Can we include a comment in `contributing/README.md` explaining why this approach was used?

Reply:
That's due to a bug with mdbook, that creates a CONTRIBUTING.html in the docs folder. But as CONTRIBUTING.md is supported within the docs folder. I'll move the file and remove this link.